kube-client/src/config/file_config.rs

```diff
@@ -525,6 +541,60 @@ impl AuthInfo {
     }
 }
 
+/// Cluster stores information to connect Kubernetes cluster used with auth plugins
+/// that have `provideClusterInfo`` enabled.
+/// This is a copy of [`kube::config::Cluster`] with certificate_authority passed as bytes without the path.
```

Comment:
this might not be possible, but if this can avoid diverging from `Cluster` then we can do a `#[serde(flatten)]`.
you could maybe call load_cert_authority after?

i guess it won't work if you need all the base64 wrapping.

Reply:
Yeah I don't think there's a "cleaner" way.. :\

Reply:
that's fine. let's at least include an upstream reference in that case. something like:

```suggestion
/// This is a copy of [`kube::config::Cluster`] with certificate_authority passed as bytes without the path.
/// Taken from [clientauthentication/types.go#Cluster](https://github.com/kubernetes/client-go/blob/477cb782cf024bc70b7239f0dca91e5774811950/pkg/apis/clientauthentication/types.go#L73-L129)
```